mkplatformfs.sh.in

```diff
@@ -117,9 +117,10 @@ case "$PLATFORM" in
     cubieboard2*|cubietruck*) PKGS="$BASEPKG ${PLATFORM%-*}-base" ;;
     odroid-u2*) PKGS="$BASEPKG ${PLATFORM%-*}-base" ;;
     odroid-c2*) PKGS="$BASEPKG ${PLATFORM%-musl}-base" ;;
-    rpi3*) PKGS="$BASEPKG rpi-base" ;;
-    rpi2*) PKGS="$BASEPKG rpi-base" ;;
-    rpi*) PKGS="$BASEPKG ${PLATFORM%-*}-base" ;;
+    rpi*) PKGS="$BASEPKG rpi-base rpi1-kernel" ;;
+    rpi2*) PKGS="$BASEPKG rpi-base rpi2-kernel" ;;
+    rpi3*) PKGS="$BASEPKG rpi-base rpi3-kernel" ;;
+    rpi4*) PKGS="$BASEPKG rpi-base rpi4-kernel" ;;
```

Comment:
This ordering with the wildcard seems to lead to the rpi1-kernel always being selected. Either rpi needs to be renamed (which the other reviewer discovered breaks other things) or the ordering of clauses need to be most to least specific instead of least to most.

Reply:
Also, `mkimage.sh` claims that the rpi4 is not a supported platform since it's not mentioned in the case statement on line 145.

Reply:
fixed